lib/dateTime.js

```diff
@@ -0,0 +1,578 @@
+			dayPeriods = ['a. m.', 'p. m.'];
+			months = [
+				['enero', 'febrero', 'marzo', 'abril', 'mayo', 'junio', 'julio', 'agosto', 'septiembre', 'octubre', 'noviembre', 'diciembre'],
+				['ene.', 'feb.', 'mar.', 'abr.', 'may.', 'jun.', 'jul.', 'ago.', 'sep.', 'oct.', 'nov.', 'dic.']
+			];
+			days = [
+				['domingo', 'lunes', 'martes', 'miércoles', 'jueves', 'viernes', 'sábado'],
+				['dom.', 'lun.', 'mar.', 'mié.', 'jue.', 'vie.', 'sáb.'],
+				['D', 'L', 'M', 'M', 'J', 'V', 'S']
+			];
+			break;
+		case 'fr':
+			dateFormats = ['dddd d MMMM yyyy', 'd MMM yyyy', 'dd/MM/yyyy', 'MMMM yyyy', 'd MMMM'];
+			months = [
+				['janvier', 'février', 'mars', 'avril', 'mai', 'juin', 'juillet', 'août', 'septembre', 'octobre', 'novembre', 'décembre'],
+				['janv.', 'févr.', 'mars', 'avr.', 'mai', 'juin', 'juil.', 'août', 'sept.', 'oct.', 'nov.', 'déc.']
+			];
+			days = [
+				['dimanche', 'lundi', 'mardi', 'mercredi', 'jeudi', 'vendredi', 'samedi'],
+				['dim.', 'lun.', 'mar.', 'mer.', 'jeu.', 'ven.', 'sam.'],
+				['D', 'L', 'M', 'M', 'J', 'V', 'S']
+			];
+			break;
+		case 'ja':
+			dateFormats = ['yyyy年M月d日', 'yyyy年M月d日', 'yyyy/MM/dd', 'yyyy年M月', 'M月d日'];
+			dayPeriods = ['午前', '午後'];
+			months[0] = months[1] = ['1 月', '2 月', '3 月', '4 月', '5 月', '6 月', '7 月', '8 月', '9 月', '10 月', '11 月', '12 月'];
+			days[0] = days[1] = days[2] = ['日', '月', '火', '水', '木', '金', '土'];
+			break;
+		case 'ko':
+			dateFormats = ['yyyy년 M월 d일 dddd', 'yyyy년 M월 d일', 'yyyy-MM-dd', 'yyyy년 M월', 'M월 d일'];
+			dayPeriods = ['오전', '오후'];
+			months[0] = months[1] = ['1월', '2월', '3월', '4월', '5월', '6월', '7월', '8월', '9월', '10월', '11월', '12월'];
+			days[0] = ['일요일', '월요일', '화요일', '수요일', '목요일', '금요일', '토요일'];
+			days[1] = days[2] = ['일', '월', '화', '수', '목', '금', '토'];
+			break;
+		case 'nl':
+			dateFormats = ['dddd d MMMM yyyy', 'd MMMM yyyy', 'dd-MM-yyyy', 'MMMM yyyy', 'd MMMM'];
+			dayPeriods = ['a.m.', 'p.m.'];
+			months = [numericMonths, numericMonths];
+			days = [
+				['zondag', 'maandag', 'dinsdag', 'woensdag', 'donderdag', 'vrijdag', 'zaterdag'],
+				['zo', 'ma', 'di', 'wo', 'do', 'vr', 'za'],
+				['Z', 'M', 'D', 'W', 'D', 'V', 'Z']
+			];
+			break;
+		case 'pt':
+			dateFormats = ['dddd, d\' de \'MMMM\' de \'yyyy', 'd\' de  \'MMMM\' de \'yyyy', 'd/M/yyyy', 'MMMM\' de \'yyyy', 'dd\' de \'MMMM'];
+			months = [
+				['janeiro', 'fevereiro', 'março', 'abril', 'maio', 'junho', 'julho', 'agosto', 'setembro', 'outubro', 'novembro', 'dezembro'],
+				['jan', 'fev', 'mar', 'abr', 'mai', 'jun', 'jul', 'ago', 'set', 'out', 'nov', 'dez']
+			];
+			days = [
+				['domingo', 'segunda-feira', 'terça-feira', 'quarta-feira', 'quinta-feira', 'sexta-feira', 'sábado'],
+				['dom', 'seg', 'ter', 'qua', 'qui', 'sex', 'sáb'],
+				['D', 'S', 'T', 'Q', 'Q', 'S', 'S']
+			];
+			break;
+		case 'sv':
+			dateFormats = ['dddd \'den\' d MMMM yyyy', 'd MMMM yyyy', 'yyyy-MM-dd', 'MMMM yyyy', 'dd MMMM'];
+			dayPeriods = ['fm', 'em'];
```

Comment:
These were `FM` and `EM` before.  I haven't been able to track down where this lives in the database, was this a conscious change to match?  We seem to have left `en` capitalized

Reply:
I was [going off this](https://github.com/andyearnshaw/Intl.js/blob/master/locale-data/json/sv-SE.json#L163) which has them lowercase. Unfortunately there's nothing in the LMS and no way to test it out since trying to format times for locales that use 24-hour clocks breaks (see #32).